scripts/quickstart.sh

```diff
@@ -8,4 +8,4 @@ kubectl apply -f <(istioctl kube-inject -f manifests/deploy-cloudant.yaml --incl
 kubectl apply -f <(istioctl kube-inject -f manifests/deploy-vote.yaml)
 kubectl apply -f <(istioctl kube-inject -f manifests/deploy-webapp.yaml)
 kubectl get nodes
-kubectl get svc istio-ingress
+kubectl get svc -n istio-system istio-ingress
```

Comment:
Do we need this here?

Reply:
For Istio 2.x, istio-ingress is in the istio-system namespace. This line is in every Kubernetes CI, so I didn't remove it

Reply:
Sure, but it wasn't in this repo before. This commit *adds* it.

Reply:
The original line is `kubectl get svc istio-ingress` which is not correct for the Istio version in this CI, so I just fix it to `kubectl get svc -n istio-system istio-ingress`